File: frontend/src/domain/enums/ProviderModels.ts
import { LLMProvider } from "./LLMProvider";

export const ProviderModels: Record<LLMProvider, string[]> = {
    [LLMProvider.Ollama]: ["llama3.2", "mistral", "qwen2.5", "gemma2", "deepseek-r1", "qwen2.5-coder", "phi4"],
};

File: backend/Models/Common/ProviderModels.cs
namespace backend.Models.Common
{
    public static class ProviderModels
    {
        public static readonly Dictionary<LLMProvider, string[]> Available = new()
        {
            [LLMProvider.Ollama] = [
                "llama3.2",
                "mistral",
                "qwen2.5",
                "gemma2",
                "deepseek-r1",
                "qwen2.5-coder",
                "phi4"
            ],
        };
    }
}